### PR TITLE
add doc for spyt operation options

### PR DESCRIPTION
**Pull request by ykuc**
[Issue](https://github.com/ytsaurus/ytsaurus-spyt/issues/18)

### CODE REVIEW

File: yt/docs/en/_includes/user-guide/data-processing/spyt/cluster/configuration.md [hunk 1]
```diff
@@ -49,6 +49,18 @@ spark-launch-yt \
...
+### Spark configuration
+
+When using `spark-launch-yt` to configure a cluster, `--params '{"spark_conf"={...};}` is available as an option:
+
+```bash
+spark-launch-yt \
+  --proxy <cluster-name> \
+  --discovery-path my_discovery_path \
+  --params '{"spark_conf"={"spark.sql.shuffle.partitions":1,"spark.cores.max":1,"spark.executor.cores"=1};}' \
+  --spark-cluster-version '1.36.0'
```
Comment (alextokarew) [inline yt/docs/en/_includes/user-guide/data-processing/spyt/cluster/configuration.md:61]: 1.36.0 is too old SPYT version, and `--spark-cluster-version` is renamed to `--spyt-version` in recent versions of SPYT

File: yt/docs/en/_includes/user-guide/data-processing/spyt/cluster/configuration.md [hunk 2]
```diff
@@ -49,6 +49,18 @@ spark-launch-yt \
   --spark-cluster-version '1.36.0'
 ```
 
+### Spark configuration
+
+When using `spark-launch-yt` to configure a cluster, `--params '{"spark_conf"={...};}` is available as an option:
```
Comment (alextokarew) [inline yt/docs/en/_includes/user-guide/data-processing/spyt/cluster/configuration.md:54]: I think it should be explained why it is needed to configure a cluster via this option.

**Review by alextokarew — CHANGES_REQUESTED** [review]

File: yt/docs/en/_includes/user-guide/data-processing/spyt/cluster/configuration.md (hunk 1 above)
Comment (ykuc) [reply yt/docs/en/_includes/user-guide/data-processing/spyt/cluster/configuration.md]: fix

**Comment by ykuc** [comment]
@alextokarew 
Should I add anything else?

**Comment by alextokarew** [comment]
> @alextokarew Should I add anything else?

Now everything is OK

**Comment by alextokarew** [comment]
Shipped!

**Pull request closed without merging**